enclave/seal.c

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include <openenclave/corelibc/stdlib.h>
+#include <openenclave/internal/crypto/gcm.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/safecrt.h>
+#include <openenclave/internal/entropy.h>
+
+static const uint8_t _iv[12] = { 0 };
```

Comment:
Why 12?

Reply:
This is the recommended IV length for GCM. The remaining 4 bytes (GHASH key is 16-byte in size) are used as the 32-bit counter.